tests/unit/test_rnn_model.py

```diff
@@ -69,10 +68,10 @@ def test_train(config_test, monkeypatch):
     """
     Check that training goes well, no errors should be thrown.
     """
-
+    preprocessor_TC = preprocessor.PreprocessorTriaxialCompression(**config_test)
     # Option 1: train using wandb
     os.system("wandb offline") # so that when you run these test the info will not be synced
-    history_wandb = train.train(config=config_test)
+    history_wandb = train.train(preprocessor_TC, config=config_test)
```

Comment:
There are quite a few places where preprocessor and config are used together. We could make config a member of preprocessor and extract config when calling train()

Reply:
Same discussion that we had with @PabRod :blush:
That is a possibility but I would keep them separated because it is possible to have two separate and independent configurations for the preprocessor and for the training.
This also offers flexibility to the user: use the same object for preprocessing the data, but different configurations for training, which is pretty common in ML task like this one.
In any case, in each of the stages (preprocessing at preprocessor, and training) the configuration is checked, naturally different keys are checked in each of the cases.